phone-store/package.json

```diff
@@ -7,7 +7,9 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "pree2e": "node node_modules/protractor/bin/webdriver-manager -- update --gecko false",
```

Comment:
I don't get what `pree2e` is doing? Is it just making sure some things are up to date? Does having this update automatically lead to possible disconnects between our (or the students') code/system and what's being run in GitHub Actions?

Reply:
This is another thing carried over from our iteration template and lab 4. I believe its needed because we are use selenium rather than direct connect (so we get the fancy element highlighting) and this is needed for it to work properly. I can't remember the exact details, sorry.

Reply:
It's not referred to anywhere, so I'm not clear on whether it matters. (I didn't understand the update stuff from the template either, TBH.) We might try removing that line and see if that breaks anything?

Reply:
Yes any `pre` or `post` prefixed scripts are automatically ran before/after the script with the same non-prefixed name.

Reply:
Gotcha. So this probably matters and needs to be kept.